cluster/saltbase/salt/fluentd-gcp/fluentd-gcp.yaml

```diff
@@ -31,6 +31,18 @@ spec:
       readOnly: true
     - name: libsystemddir
       mountPath: /host/lib
+    livenessProbe:
+      initialDelaySeconds: 600
+      periodSeconds: 60
```

Comment:
In the worst case, delay between the moment a problem occurs and the detection is `period + threshold`. 6 minutes is better than 10, right? This probe is very lightweight anyway.